core/scss/components/main-nav/_main-nav--light.scss

```diff
@@ -0,0 +1,165 @@
+
+.su-main-nav.su-main-nav--light {
+
+  .su-main-nav__toggle {
+    &[aria-expanded="true"] {
+      &::before {
+        background-color: $color-driftwood;
+      }
+
+      &:hover {
+        &::before {
+          background-color: $color-dark-red;
+        }
+      }
+    }
+  }
+
+  ul {
+    background-color: $color-white;
+  }
+
+  > ul {
+
+    @include grid-media('lg') {
+      background-color: transparent;
+
+      > li {
+        > a {
+          &:hover,
+          &:focus {
+            color: $color-dark-red;
+
+            &::before {
+              background-color: $color-dark-red;
+            }
+          }
+
+          &:active {
+            color: $color-black;
+
+            &::before {
+              background-color: $color-black;
+            }
+          }
+
+          &[aria-expanded="true"] {
+            color: $color-black;
+
+            &::before {
+              background-color: $color-driftwood;
+            }
+
+            &:hover {
+              color: $color-dark-red;
+
+              &::before {
+                background-color: $color-dark-red;
+              }
+            }
+          }
+        }
+
+        &.su-main-nav__item--current {
+          > a {
+            color: $color-black;
+
+            &::before {
+              background-color: $color-driftwood;
+            }
+
+            &:hover,
+            &:focus {
+              color: $color-dark-red;
+
+              &::before {
+                background-color: $color-dark-red;
+              }
+            }
+
+          }
+        }
+      }
+    }
+  }
+
+  li {
+    a {
+      color: $color-black;
+      border-bottom-color: #d9d9d9;
+
+      &:hover,
+      &:focus {
+        &::before {
+          background-color: $color-black;
+        }
+      }
+
+      &:active {
+        color: $color-dark-red;
+
+        &::before {
+          background-color: $color-dark-red;
+        }
+      }
+
+      &[aria-expanded="true"] {
+        &::before {
+          background-color: $color-dark-red;
+        }
+      }
+    }
+
+    &.su-main-nav__item--current {
+      > a {
+        color: $color-dark-red;
+
+        &::before {
+          background-color: $color-dark-red;
+        }
+      }
+    }
+
+    &.su-main-nav__item--parent {
+      > a {
+        &::after {
+          background: url("#{$image-path}/plus-black.svg") no-repeat 0 0;
+          background-size: 100%;
+          content: "";
+          display: inline-block;
+          position: absolute;
+          right: 1em;
+          top: 1.03em;
+          height: .9em;
+          width: .9em;
+
+          @include grid-media('lg') {
+            background: url("#{$image-path}/caret-down-black.svg") no-repeat 0 0;
+            background-size: 100%;
+            position: relative;
+            right: 0;
+            top: 0;
+            height: 11px; //small svg gets chopped off if use em instead of px
+            width: 11px;
+            transition: all .3s ease-out;
```

Comment:
It is recommended to avoid using `transition: all` and to use specific property transitions. 
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x